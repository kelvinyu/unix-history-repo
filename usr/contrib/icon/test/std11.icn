record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("integer(2) ----> ",image(integer(2)) | "none")
   write("integer(2.0) ----> ",image(integer(2.0)) | "none")
   write("integer(2.7) ----> ",image(integer(2.7)) | "none")
   write("integer(\"2\") ----> ",image(integer("2")) | "none")
   write("integer(\" 2\") ----> ",image(integer(" 2")) | "none")
   write("integer(\"2 \") ----> ",image(integer("2 ")) | "none")
   write("integer(\"+2\") ----> ",image(integer("+2")) | "none")
   write("integer(\"-2\") ----> ",image(integer("-2")) | "none")
   write("integer(\"- 2\") ----> ",image(integer("- 2")) | "none")
   write("integer(\" -    2 \") ----> ",image(integer(" -    2 ")) | "none")
   write("integer(\"\") ----> ",image(integer("")) | "none")
   write("integer(\"--2\") ----> ",image(integer("--2")) | "none")
   write("integer(\" \") ----> ",image(integer(" ")) | "none")
   write("integer(\"-\") ----> ",image(integer("-")) | "none")
   write("integer(\"+\") ----> ",image(integer("+")) | "none")
   write("integer(\".\") ----> ",image(integer(".")) | "none")
   write("integer(\".3\") ----> ",image(integer(".3")) | "none")
   write("integer(\"0.3\") ----> ",image(integer("0.3")) | "none")
   write("integer(\" . 3\") ----> ",image(integer(" . 3")) | "none")
   write("integer(\"e2\") ----> ",image(integer("e2")) | "none")
   write("integer(\"22222222222222222222222222222222222222222222222222222222222\") ----> ",image(integer("22222222222222222222222222222222222222222222222222222222222")) | "none")
   write("integer(\"3e500\") ----> ",image(integer("3e500")) | "none")
   write("integer(\"7r4\") ----> ",image(integer("7r4")) | "none")
   write("integer(\"4r7\") ----> ",image(integer("4r7")) | "none")
   write("integer(\"4r 7\") ----> ",image(integer("4r 7")) | "none")
   write("integer(\"7r 4\") ----> ",image(integer("7r 4")) | "none")
   write("integer(\"16rff\") ----> ",image(integer("16rff")) | "none")
   write("integer(\"36rcat\") ----> ",image(integer("36rcat")) | "none")
   write("integer(\"36Rcat\") ----> ",image(integer("36Rcat")) | "none")
   write("integer(\"36rCAT\") ----> ",image(integer("36rCAT")) | "none")
   write("integer(\"1r1\") ----> ",image(integer("1r1")) | "none")
   write("integer(\"2r0\") ----> ",image(integer("2r0")) | "none")
   write("integer(integer) ----> ",image(integer(integer)) | "none")
   write("integer := abs ----> ",image(integer := abs) | "none")
   write("integer(\"a\") ----> ",image(integer("a")) | "none")
end
