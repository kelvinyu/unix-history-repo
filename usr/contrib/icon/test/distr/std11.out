integer(2) ----> 2
integer(2.0) ----> 2
integer(2.7) ----> 2
integer("2") ----> 2
integer(" 2") ----> 2
integer("2 ") ----> 2
integer("+2") ----> 2
integer("-2") ----> -2
integer("- 2") ----> none
integer(" -    2 ") ----> none
integer("") ----> none
integer("--2") ----> none
integer(" ") ----> none
integer("-") ----> none
integer("+") ----> none
integer(".") ----> none
integer(".3") ----> 0
integer("0.3") ----> 0
integer(" . 3") ----> none
integer("e2") ----> none
integer("22222222222222222222222222222222222222222222222222222222222") ----> none
integer("3e500") ----> none
integer("7r4") ----> 4
integer("4r7") ----> none
integer("4r 7") ----> none
integer("7r 4") ----> none
integer("16rff") ----> 255
integer("36rcat") ----> 15941
integer("36Rcat") ----> 15941
integer("36rCAT") ----> 15941
integer("1r1") ----> none
integer("2r0") ----> 0
integer(integer) ----> none
integer := abs ----> function abs
Run-time error 102 at line 43 in std11.icn
numeric expected
offending value: "a"
